homeassistant/components/gandi_livedns/__init__.py

```diff
@@ -0,0 +1,114 @@
+"""Integrate with Gandi Live DNS service."""
+from datetime import timedelta
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_DOMAIN,
+    CONF_NAME,
+    CONF_TIMEOUT,
+    CONF_TTL,
+    CONF_TYPE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import (
+    CONF_IPV6,
+    CONF_UPDATE_INTERVAL,
+    DATA_UPDATE_INTERVAL,
+    DOMAIN,
+    SERVICE_UPDATE_RECORDS,
+)
+from .gandi import GandiApiLiveDNS
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = cv.deprecated(DOMAIN)
+
```

Comment:
This is a new integration and thus cannot have a deprecated YAML configuration 😉 

```suggestion
```

Reply:
I have removed this :)